starboard/media.h

```diff
@@ -28,6 +28,8 @@
 extern "C" {
 #endif
 
+// This is not a comment.
```

Comment:
Looks like a comment to me. What do we need this for ? :)

Reply:
This was a test change to see if it would trigger targets depending on starboard (it did).